FinSearchUnified/Bundle/SearchBundleFindologic/ConditionHandler/SimpleConditionHandler.php

```diff
@@ -3,14 +3,16 @@
 namespace FinSearchUnified\Bundle\SearchBundleFindologic\ConditionHandler;
 
 use FinSearchUnified\Bundle\SearchBundleFindologic\ConditionHandlerInterface;
-use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder;
+use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder\NewQueryBuilder;
 use Shopware\Bundle\SearchBundle\Condition\SimpleCondition;
 use Shopware\Bundle\SearchBundle\ConditionInterface;
 use Shopware\Bundle\StoreFrontBundle\Struct\ShopContextInterface;
 
+/**
+ * @deprecated
```

Comment:
Why is this deprecated?

If you want to introduce deprecation notices, you should always give alternatives or give a really good reason to why something is deprecated. Just telling someone that this is deprecated won't help them in anyway.

Reply:
The `SimpleCondition` is deprecated which we are using from the default Shopware namespace. So it makes sense to deprecate the handler as well.

Reply:
Shopwares `SimpleCondition` may be deprecated, but we still want to have this handler.

```php
/**
 * @deprecated in 5.6, will be removed in 5.7, without replacement
 */
```

Only because this is deprecated in 5.7, does not mean that we still can have the handler. It just uses a different condition class internally, we definitely won't remove this handler.

Reply:
Alright.